SPECS-EXTENDED/libvirt-java/libvirt-java.spec

```diff
@@ -4,7 +4,7 @@ Summary:    Java bindings for the libvirt virtualization API
 Name:       libvirt-java
 Version:    0.4.9
 Prefix:     libvirt
-Release:    17%{?dist}
+Release:    18%{?dist}
```

Comment:
May I ask you to revert the changes in this spec? I'm already migrating this file to `SPECS` [here](https://github.com/microsoft/CBL-Mariner/pull/1845) and adding some additional changes to it anyway.

Reply:
Done, thanks!